src/Common/nunit/CommandLineOptions.cs

```diff
@@ -296,6 +298,23 @@ protected virtual void ConfigureOptions()
             this.Add("where=", "Test selection {EXPRESSION} indicating what tests will be run. See description below.",
                 v => WhereClause = RequiredValue(v, "--where"));
 
+            this.Add("params|D=", "Define a test parameter.",
```

Comment:
@oznetmaster had asked for -D but I guess you're right. In addition to being the wrong letter, it's the first one where we would be using a capital letter. No reason to do that. I suggest we either use `params|p=` or `define|d=`

@oznetmaster do you want to weigh in on this?